libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -1137,7 +1149,8 @@ QDomDocument ProfileManagerPrivate::constructProfileDocument(const Profile &aPro
 bool ProfileManagerPrivate::writeProfileFile(const QString &aProfilePath,
         const QDomDocument &aDoc)
 {
-    //FUNCTION_CALL_TRACE;
+    FUNCTION_CALL_TRACE;
+    LOG_WARNING("writeProfileFile() called, forcing disk write:" << aProfilePath);
```

Comment:
While looking at this code, is this safe to do to a file IO: if (file.open(QIODevice::WriteOnly | QIODevice::Truncate)), what is the main advantage of a truncate to a just open created QFile ?